### PR TITLE
Add documentation page for Instant

### LINKED ISSUE #685: No mention of Instant in manual - intended?
The [manual](https://js-joda.github.io/js-joda/manual/) has entries for LocalDate, LocalTime, LocalDateTime, ZonedDateTime, Period, and Duration, but no entry for Instant. This seems like a strange omission, as Instant is the most appropriate type for representing timestamps in a log file (a rather common use-case). There's good documentation at https://js-joda.github.io/js-joda/class/packages/core/src/Instant.js~Instant.html — why not copy that introductory material into a `docs/manual/Instant.html` file? Or is the omission of Instant from the manual actually deliberate?

### PR DESCRIPTION
**Pull request by alisabzevari**
to resolve #685

### CODE REVIEW

**Review by pithu — COMMENTED** [review]
Hi @alisabzevari, thanks a lot, great work, i just have some comments

File: esdoc/manual/Instant.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+An `Instant` is an instantaneous point on the time-line measured from the epoch of 1970-01-01T00:00:00Z in epoch-seconds and nanosecond-of-second.
+
+### Create an `Instant`
+
+```javascript
+// Obtain the current instant from the system clock
+Instant.now();
+
+// Obtain the current instant from the UTC clock
+Instant.now(Clock.systemUTC());
```
[the remark this answers is not in the document]
Comment (alisabzevari) [reply esdoc/manual/Instant.md:12]: I wanted to show that instant also accepts the clock instant. Do you prefer to remove it completely or can I create a clock instance an use it there?

**Comment by alisabzevari** [comment]
I have applied all the comments. Please have a look. Sorry for the late reply.

**Review by pithu — APPROVED** [review]
LGTM, thanks for contributing

**Pull request merged**